Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyIntCtrlCommon.h

```diff
@@ -169,6 +169,10 @@ namespace AzToolsFramework
         {
             AzToolsFramework::PropertyEditorGUIMessages::Bus::Broadcast(&PropertyEditorGUIMessages::Bus::Events::RequestWrite, newCtrl);
         });
+        this->connect(newCtrl, &PropertyControl::editingFinished, this,[newCtrl]()
```

Comment:
uber nit: extra space `this,[newCtrl]` -> `this, [newCtrl]`

Reply:
clang-format will fix this for you if you highlight the lines and hit `Ctrl-K, F` (`Ctrl-K, D` will format the whole file which could be done as a separate change potentially)